scalanet/test/src/io/iohk/scalanet/peergroup/TCPPeerGroupCachedUntilConnectSpec.scala

```diff
@@ -0,0 +1,60 @@
+package io.iohk.scalanet.peergroup
```

Comment:
I didn't understand what this file is testing. Is it a feature not covered in the main TCPPeerGroupSpec?